src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets

```diff
@@ -0,0 +1,305 @@
+      <_BlazorAssembliesToLink Include="@(ReferenceCopyLocalPaths->WithMetadataValue('Extension','.dll')->'-a &quot;%(FullPath)&quot;')" />
+      <_BlazorAssembliesToLink Include="@(IntermediateAssembly->'-a &quot;%(FullPath)&quot;')" />
+      <_BlazorFolderLookupPaths Include="@(MonoBaseClassLibraryFolder->'-d &quot;%(Identity)&quot;')" />
+      <_BlazorAssemblyDescriptorFiles
+        Include="@(BlazorLinkerDescriptor->'-x &quot;%(FullPath)&quot;')" Condition="'@(BlazorLinkerDescriptor)' != ''" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <_BlazorMonoLinkerAdditionalOptions>$(AdditionalMonoLinkerOptions)</_BlazorMonoLinkerAdditionalOptions>
+    </PropertyGroup>
+
+    <!-- Clear the contents of /obj/debug/netstandard2.0/blazor/blazor/linker -->
+    <Delete Files="$(BlazorIntermediateLinkerOutputPath)*.dll" />
+
+    <!-- Run the linker and put the results in /obj/debug/netstandard2.0/blazor/blazor/linker -->
+    <Exec Command="dotnet $(MonoLinkerPath) $(_BlazorMonoLinkerAdditionalOptions) @(_BlazorFolderLookupPaths, ' ') -o $(BlazorIntermediateLinkerOutputPath) @(_BlazorAssemblyDescriptorFiles, ' ') @(_BlazorAssembliesToLink, ' ')"  />
+
+    <!-- Collect the contents of /obj/debug/netstandard2.0/blazor/blazor/linker/ -->
+    <ItemGroup>
+      <_BlazorLinkerOutput Include="$(BlazorIntermediateLinkerOutputPath)*.dll" />
+    </ItemGroup>
+
+    <!-- 
+    Write the list of files in /obj/debug/netstandard2.0/blazor/blazor/linker/ into 
+    /obj/debug/netstandard2.0/blazor/blazor/linker.marker.txt
+    -->
+    <WriteLinesToFile File="$(BlazorIntermediateLinkerResultFilePath)" Lines="@(_BlazorLinkerOutput)" Overwrite="true" />
+
+    <!-- Add /obj/debug/netstandard2.0/blazor/blazor/linker.marker.txt to the list of written files. -->
+    <ItemGroup>
+      <FileWrites Include="$(BlazorIntermediateLinkerResultFilePath)" />
+    </ItemGroup>
+  </Target>
+
+  <Target 
+    Name="CollectLinkerOutputs"
+    Condition="'$(BlazorShouldRunMonoLinker)' == 'true'"
+    DependsOnTargets="RunMonoLinker">
+    <!--
+    Read the outputs from the linker (from this run or a previous run) and set them in an item group for
+    later use.
+    -->
+    <ReadLinesFromFile File="$(BlazorIntermediateLinkerResultFilePath)">
+      <Output TaskParameter="Lines" ItemName="_OptimizedFiles"/>
+    </ReadLinesFromFile>
+    
+    <ItemGroup>
+      <BlazorItemSpec Include="@(_OptimizedFiles)">
+        <TargetOutputPath>$(ProjectDir)$(OutputPath)$(BaseBlazorRuntimeBinOutputPath)%(FileName)%(Extension)</TargetOutputPath>
+        <Type>Assembly</Type>
+        <PrimaryOutput Condition="'%(FileName)' == @(IntermediateAssembly->'%(FileName)')">true</PrimaryOutput>
+      </BlazorItemSpec>    
+      <FileWrites Include="@(BlazorItemSpec->WithMetadataValue('Type','Assembly')->'%(TargetOutputPath)')" />
+    </ItemGroup>
+  
+  </Target>
+
+  <Target
+    Name="ResolveRuntimeDependencies"
+    Condition="'$(BlazorShouldRunMonoLinker)' == 'false'"
```

Comment:
Unless you can guarantee that this property always has the value `true` or `false` then this should be `==''`. There aren't booleans in MSBuild 😆

Reply:
I do understand, I've changed it. But using '' and not '' also has its caveheats. Its easy enough to guarantee by just initializing '' to false